src/components/app/App.js

```diff
@@ -16,44 +17,46 @@ const App = () => {
   return (
     <ThemeContextProvider>
       <ShowContextProvider>
-        <Navbar />
-        <Switch>
-          <Route
-            exact
-            path="/"
-            render={() => {
-              return (
-                <div className="years-view">
-                  <YearsContainer />
-                </div>
-              );
-            }}
-          />
-          <Route
-            exact
-            path="/years/:year"
-            render={({ match }) => {
-              return (
-                <div className="app">
-                  <ShowsContainer year={match.params.year} />
-                </div>
-              );
-            }}
-          />
-          <Route
-            exact
-            path="/shows/:id"
-            render={({ match }) => {
-              return (
-                <div className="app">
-                  <TracksContainer id={match.params.id} type="tracks" />
-                </div>
-              );
-            }}
-          />
-          <Route render={() => <p>Error. Page Not Found!</p>} />
-        </Switch>
-        <Footer />
+        <div className="app-wrapper">
+          <Navbar />
+          <Switch>
+            <Route
+              exact
+              path="/"
+              render={() => {
+                return (
+                  <div className="app">
```

Comment:
I think that changing this className to container may be a little more dev empathetic.